Stripe/STPBankAccount.m

```diff
@@ -49,31 +56,21 @@ + (STPBankAccountStatus)statusFromString:(NSString *)string {
     return STPBankAccountStatusNew;
 }
 
-+ (NSString *)stringFromStatus:(STPBankAccountStatus)status {
++ (nullable NSString *)stringFromStatus:(STPBankAccountStatus)status {
```

Comment:
Ah yeah good point. Theoretically as long as you only pass in valid enum values it will always be nonnull, I'll fix.